velox/docs/functions/presto/decimal.rst

```diff
@@ -159,6 +160,25 @@ then divide by B and round to the nearest whole. This method works as long as
 rescale factor does not exceed 38. If :code:`s + s2 - s1` exceeds 38, an error
 is raised.
 
+For decimal modulus operation :code:`a % b`, Presto calculates the result
+precision and scale as follows:
+
+::
+
+    p = min(p2 - s2, p1 - s1) + max(s1, s2)
```

Comment:
It would be nice to explain where this formula comes from. It is not obvious.

If a and b are integers and r = a % b, then r < a && r < b, hence, the number of digits needed to represent r is no more than min of number of digits needed to represent a or b. 

When a and b are decimals, we compute mod by (1) rescaling a and b to make them integers: a * 10 ^ s, b * 10^ s; (2) computing mod of integers; (3) reporting the results using s scale. Hence, the number of digits needed to represent the result is s + min(p2 - s2, p1 - s1).

Is this how it goes?